pimd/pim_oil.h

```diff
@@ -166,7 +166,8 @@ static inline bool oil_if_has(struct channel_oil *c_oil, mifi_t ifi)
 	return !!IF_ISSET(ifi, &c_oil->oil.mf6cc_ifset);
 }
 
-static inline void oil_if_set(struct channel_oil *c_oil, mifi_t ifi, bool set)
+static inline void oil_if_set(struct channel_oil *c_oil, mifi_t ifi,
+			      uint8_t set)
```

Comment:
false was being passed to oil_if_set in pim_channel_del_oif, we need to change there as well.

Reply:
go for it then.